utils/utils.go

```diff
@@ -0,0 +1,261 @@
+package utils
+
+// #include "limits.h"
+import "C"
+
+import (
+	"golang.org/x/sys/unix"
+	"net"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/gluster/glusterd2/errors"
+)
+
+const testXattr = "trusted.glusterfs.test"
+const volumeIDXattr = "trusted.glusterfs.volume-id"
+const gfidXattr = "trusted.gfid"
+
+// PosixPathMax represents POSIX_PATH_MAX
+var PosixPathMax int
```

Comment:
You could just do `var PosixPathMax int = C._POSIX_PATH_MAX` instead of having an `init` function. Better still make it a `const`.